deploy/kube-config/influxdb/grafana.yaml

```diff
@@ -23,7 +23,7 @@ spec:
         env:
         - name: INFLUXDB_HOST
           value: monitoring-influxdb
-        - name: GRAFANA_PORT
+        - name: GF_SERVER_HTTP_PORT
```

Comment:
Updated `grafana/run.sh` so it doesn't export the `GF_SERVER_HTTP_PORT` environment variable. By default Grafana should bind to port 3000.